### PR TITLE
Delete line number change assert

### PR DESCRIPTION
**Pull request by zurk**

### CODE REVIEW

File: lookout/core/bytes_to_unicode_converter.py [hunk 1]
```diff
@@ -82,15 +82,14 @@ def _convert_position(self, byte_position: bblfsh.Position) -> bblfsh.Position:
         offset = self._byte_to_str_offset[byte_position.offset]
         line_num = numpy.argmax(self._lines_offset > offset) - 1
         col = offset - self._lines_offset[line_num]
+        # line number can be changed. File example:
```
Comment (vmarkovtsev) [inline lookout/core/bytes_to_unicode_converter.py]:
```suggestion
        # line number can change. File example:
```

File: lookout/core/bytes_to_unicode_converter.py [hunk 2]
```diff
@@ -82,15 +82,14 @@ def _convert_position(self, byte_position: bblfsh.Position) -> bblfsh.Position:
...
+        # github.com/src-d/style-analyzer/blob/ed9324d783eb0082d2f59de336190c8805a33c75/lookout/style/format/tests/bugs/002_bad_line_positions/browser-policy-content.js
         line = self._lines[line_num]
         if len(line) == col:
             if line.splitlines()[0] != line:
                 # ends with newline
                 line_num += 1
                 col = 0
-        assert line_num + 1 == byte_position.line, \
```
Comment (vmarkovtsev) [inline lookout/core/bytes_to_unicode_converter.py]: I guess that it may be bigger, but never smaller. Shall we change this assertion to check `>=`?

Comment (zurk) [reply lookout/core/bytes_to_unicode_converter.py]: let's try and see :)

Comment (zurk) [reply lookout/core/bytes_to_unicode_converter.py]: At least test in style-analyzer is passed.

File: lookout/core/bytes_to_unicode_converter.py [hunk 3]
```diff
@@ -82,15 +82,17 @@ def _convert_position(self, byte_position: bblfsh.Position) -> bblfsh.Position:
...
-            ("Line number is changed from %d to %d when convert from byte position to string "
-             "position.") % (byte_position.line, line_num + 1)
+        assert line_num + 1 >= byte_position.line, \
+            ("Unicode Line number %d is smaller then in bytes (%d)."
```
Comment (vmarkovtsev) [inline lookout/core/bytes_to_unicode_converter.py]: Line -> line

**Pull request merged**